.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - run:
           name: run tests
           command: |
-            xvfb-run pytest --cov=onionshare --cov=onionshare_gui --cov-report=term-missing -vvv tests/
+            xvfb-run pytest --cov=onionshare --cov=onionshare_gui --cov-report=term-missing -vvv tests/ --rungui
```

Comment:
minor nitpick: it's more readable to have the flag `--rungui` on the left side of `tests/` with the other flags. But it doesn't seem to impact the actual command.